app/classifier/index.cjsx

```diff
@@ -414,6 +416,15 @@ Classifier = React.createClass
   toggleExpertClassification: (value) ->
     @setState showingExpertClassification: value
 
+  warningToggleOn: ->
+    @setState backButtonWarning: true
+
+  warningToggleOff: ->
+    @setState backButtonWarning: false 
+
+  renderBackButtonWarning: ->
+    <p className={"back-button-warning"} >{"Going back will delete all annotations for the current task."}</p>
```

Comment:
I'm not sure that you need the curly brackets around the class name and paragraph text here.